verticles/graphql/src/main/java/com/gentics/mesh/graphql/type/MicroschemaTypeProvider.java

```diff
@@ -116,13 +108,7 @@ public GraphQLObjectType createType() {
 		GraphQLOutputType type = GraphQLList.list(fieldListBuilder.build());
 
 		// .fields
-		schemaType.field(newFieldDefinition().name("fields").type(type).dataFetcher(env -> {
-			List<FieldSchema> fields = loadModelWithFallback(env).getFields();
-			if (fields.isEmpty()) {
-				fields = Collections.singletonList(emptySchemaFieldDummy("This microschema has no fields. Do not use this filter."));
-			}
-			return fields;
-		}));
+		schemaType.field(newFieldDefinition().name("fields").type(type).dataFetcher(env -> loadModelWithFallback(env).getFields()));
```

Comment:
This is safe to remove, since we return a list of not the GraphQLType, but FieldSchema, which is legal.